modules/virt-about-live-migration.adoc

```diff
@@ -7,14 +7,20 @@
 [id="virt-about-live-migration_{context}"]
 = About live migration
 
-Live migration is the process of moving a running virtual machine instance (VMI) to another node in the cluster without interrupting the virtual workload or access. If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates when the node that the VMI runs on is placed into maintenance mode. You can also manually start live migration by selecting a VMI to migrate.
+Live migration is the process of moving a running virtual machine instance (VMI) to another node in the cluster without interrupting the virtual workload or access. If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates when the node that the VMI runs on is placed into maintenance mode.
 
-You can use live migration if the following conditions are met:
+You can manually start live migration by selecting a VMI to migrate.
 
-* Virtual machines must have a persistent volume claim (PVC) with a shared `ReadWriteMany` (RWX) access mode to be live migrated.
+Live migration has the following requirements:
 
-* The pod network binding must not be of the bridge interface type `()`.
+* Shared storage with `ReadWriteMany` (RWX) access mode
 
-* Live migration is supported for virtual machines that are attached to an SR-IOV network interface.
+* Sufficient RAM and network bandwidth
 
-* If the virtual machine uses a host model CPU, live migration is supported only between nodes that support the virtual machine's host model CPU.
+* Appropriate CPUs with sufficient capacity on the worker nodes
++  
+If the CPUs have different capacities, live migration might be very slow or fail.
```

Comment:
Sorry for not catching this on the first review. I'm not sure about this point. CPUs on different workers would usually have different capacities (maybe one is used on 90 percent and the other 95), but that should not affect the migration. What could affect migration is if the destination does not have sufficient capacity, but that is already covered in the item above. I think we should either drop this or expand on what we mean by different capacities and when it could negatively affect the migration. That being said, I'm no expert in live-migration, so if there is a hunch that this item is needed, we may need to call in somebody from compute.

Reply:
IIRC, Dan Kenigsberg added this. I'll ask Adam.

Reply:
@stu-gott 
What do you think?

Reply:
I concur with @phoracek. I can't understand the origin or meaning of this statement, and as far as I can see it is redundant with the previous line.

Reply:
Sorry I just realized I might have been unclear. I think we should we remove this line.